schemas/1.5/dbt_yml_files-1.5.json

```diff
@@ -254,6 +257,9 @@
               "additionalProperties": false
             }
           },
+          "group": {
+            "type": "string"
+          },
```

Comment:
In the dbt_project files, group is def'd once and everywhere else it uses refs, which is good. Any reason you haven't done the same with the dbt_yml_files for 1.5 and 1.6?